layers/sip.go

```diff
@@ -0,0 +1,360 @@
+// Copyright 2017 Google, Inc. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file in the root of the source
+// tree.
+
+package layers
+
+import (
```

Comment:
Imports should be first the golang packages, a blank line and the gopacket package